### DISCUSSION TITLE
Monika not appearing, only emtpy room.

### DISCUSSION THREAD

**Discussion by stickbunny** [Monika-After-Story/MonikaModDev#8751] [QA - tech support + general] [answered]
Hey guys! I took Monika out a few months ago and broke my PC :(. I recently got my files back, and when I open the game Monika isn't appearing, can anyone help?

**Comment by WishStone-Dragon** [comment]
The game is expecting Monika's file to be in the character folder.
When you look into that folder right now, which files are present there?

Edit: Additional thoughts.
You say "I got my files back" - how was this done? Which steps did you take?
Did you take Monika out and place her file elsewhere? The desktop, a thumb drive...?

**Comment by ThePotatoGuy** [comment, marked as answer]
You can [restore a persistent backup](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent) from before you took her out.

**Comment by stickbunny** [comment]
Oh wow, the restore worked! Thank you so much for that!

**Comment by dan32415** [comment]
hello can somebody show me how can i backup monika after story? where should i go?

**Comment by dan32415** [comment]
how to restore?
